Tribler/community/multichain/database.py

```diff
@@ -189,23 +189,35 @@ def check_database(self, database_version):
 
         return LATEST_DB_VERSION
 
-    def total_up(self, public_key):
-        """
-        Gets the total uploaded value from the focus.
-        :param public_key: public key of the focus node
-        :return: number representing the amount of uploaded data
-        """
-        block = self.get_latest(public_key)
-        return block.total_up if block else 0
-
-    def total_down(self, public_key):
-        """
-        Gets the total downloaded value from the focus.
-        :param public_key: public key of the focus node
-        :return: number representing the amount of uploaded data
-        """
-        block = self.get_latest(public_key)
-        return block.total_down if block else 0
+    # def total_up(self, public_key):
+    #     """
+    #     Gets the total uploaded value from the focus.
+    #     :param public_key: public key of the focus node
+    #     :return: number representing the amount of uploaded data
+    #     """
+    #     query = u"SELECT sum_"
+    #     block = self.get_latest(public_key)
+    #     return block.total_up if block else 0
+    #
+    # def total_down(self, public_key):
+    #     """
+    #     Gets the total downloaded value from the focus.
+    #     :param public_key: public key of the focus node
+    #     :return: number representing the amount of uploaded data
+    #     """
+    #     block = self.get_latest(public_key)
+    #     return block.total_down if block else 0
+
+    def total_traffic(self, public_key):
```

Comment:
Please add documentation explaining what this method does... what the parameters are, what it returns etc...